### ISSUE TITLE
gentoo ebuild

### ISSUE THREAD

**Issue by repomansez** [slavfox/Cozette#128]
hi!

I have written a Gentoo ebuild for cozette-otb, should you want to list it as an alternative Linux install method, it's in my personal overlay:

https://github.com/repomansez/cute-overlay/blob/master/media-fonts/cozette/cozette-1.24.1.ebuild

**Comment by chi-xigua** [comment]
Very nice, but some changes could improve it

you can change the SRC_URI from:
```bash
SRC_URI="https://github.com/slavfox/Cozette/releases/download/v.${PV}/CozetteFonts-v-1-24-1.zip"
```
to:
```bash
SRC_URI="https://github.com/slavfox/Cozette/releases/download/v.${PV}/CozetteFonts-v-${PV//./-}.zip"
```

so changing the version is as simple as changing the ebuild name


and here in your ebuild:
```bash
DEPEND="app-arch/unzip"
RDEPEND="${DEPEND}"
BDEPEND=""
```
you don't need unzip as a runtime dependency, so you should move the `${DEPEND}` to the BDEPEND variable instead for build time dependency 
```bash
DEPEND="app-arch/unzip"
RDEPEND=""
BDEPEND="${DEPEND}"
```

**Comment by repomansez** [comment]
@chi-xigua you're totally right! No idea why I made it like that hehe. But seems like @slavfox isn't interested in mentioning this :(
